### ISSUE TITLE
Explain connection between freezable tasks and freezable task sources

### ISSUE THREAD

**Issue by domenic** [WICG/web-lifecycle#16]
I think the idea is that if a task source is freezable, all tasks queued on it are automatically freezable? Right now the spec doesn't explain.

If this is the case, maybe we don't need to have a concept of a freezable task at all, and we can just say to ignore tasks whose task source is freezable?

Or do we need the flexibility to sometimes make a freezable task, even on an unfreezable task source?

**Comment by spanicker** [comment]
Yes that makes sense, I've removed the "modification" for freezable task, and added a line to task source:
"All tasks queued on freezable task sources are automatically freezable"
